contents/handbook/people/compensation.mdx

```diff
@@ -59,6 +59,12 @@ With exception of team members at the very beginning of their career, we hire in
 
 The definition of what is needed to progress from one step to the next in more detail depends on your role. Ask your manager for detail of what you need to work on.
 
+### Compensation reviews
+
+We conduct these for the whole team every 3 months. As we do these much more frequently than regular companies, team members should definitely not expect these to result in a change to their Step or Level each time - often they will stay the same. Additionally, team members will find that their Step will change more frequently than their Level.
```

Comment:
I'd add that we may change pay _without_ changing step or level if we feel market rates have changed upwards

Reply:
1) I'd probably add "roughly" as I think it's better not to create a stressful moment every 3 months, to not set a clear expectation that will nearly always be not met. Of course, you and Eltje can schedule this whenever you want. I can also see a scenario where it could be _more_ frequent because in the process of hiring a role, we'll realize the benchmark isn't quite appropriate.

2) I'd probably add something to clarify what to expect from a team member perspective - this would leave me wondering do I need to have a showdown type meeting ie:

"We conduct these for the whole team roughly every 3 months. You do not need to do anything - our goal is to keep your compensation at an appropriate level without you having to ask."